### PR TITLE
Ignatii/develop api

### PR DESCRIPTION
**Pull request by IgnAhT**
Created 'add_document' view with a corresponding test block. URL path is hooked up to 'add/'.

### CODE REVIEW

**Review by ni-sauvage — APPROVED** [review]
I'm happy to merge this, test-cases and functionality look good!

File: api/models.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Block(models.Model):
     The text content of a block.
     """
     
-    block_order = models.IntegerField(default=0, unique=True)
+    block_order = models.IntegerField(default=0)
```
Comment (MaxCunningham19) [inline api/models.py:24]: Q: Should this not still have unique = True?

File: api/urls.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 app_name = 'api'
 
 urlpatterns = [
     path('document/', DocumentList.as_view(), name='get_documents'),
     path('document/<int:pk>', get_document_blocks, name='get_document_blocks'),
-    path('check/document/<int:pk>', check_document_blocks, name='check_document_blocks')
-]
+    path('check/document/<int:pk>', check_document_blocks, name='check_document_blocks'),
+    path('add/', add_documents, name='add_documents')]
```
Comment (MaxCunningham19) [inline api/urls.py:10]: This should just be documents/ according to the [rest convention](https://restfulapi.net/resource-naming/)

File: api/views.py [hunk 3]
```diff
@@ -17,11 +20,24 @@ class DocumentDetail(generics.RetrieveAPIView):
 def get_document_blocks(request, pk):
     blocks = Block.objects.filter(block_document=pk)
     serializer = BlockSerializer(blocks, many=True)
-    return rest_framework.response.Response(serializer.data, status=200)
+    return Response(serializer.data, status=200)
```
Comment (MaxCunningham19) [inline api/views.py:23]: I think there should be a check if the serialised is correct incase the document requested doesn't exist

**Review by MaxCunningham19 — CHANGES_REQUESTED** [review]

**Review by MaxCunningham19 — APPROVED** [review]
LGTM

**Pull request merged**